quickwit-proto/proto/search_api.proto

```diff
@@ -167,3 +170,47 @@ message FetchDocsResult {
   // List of complete hits.
   repeated Hit hits = 1;
 }
+
+
+// -- Export -------------------
+
+
+message ExportRequest {
+  // Index ID
+  string index_id = 1;
+
+  // Query
+  string query = 2;
+
+  // Fields to search on
+  repeated string  search_fields = 3;
+  
+  // Time filter
```

Comment:
Add comment:
The time filter is interpreted as a semi-open interval. [start, end)